### DISCUSSION TITLE
Funcube Dongle Pro +

### DISCUSSION THREAD

**Discussion by VITango** [AlexandreRouma/SDRPlusPlus#890] [Ideas]
hey Alex, awesome app, well done and works well on my old T5 Samsung tablet. 
Any chance you have looked at making it compatable to use the Funcube Dongle Pro + in the list of SDR devices 🙏
Cheers

**Comment by ghost** [comment]
Hello,
You can use the pothosware/SoapyFCDPP driver for SoapySDR.  This makes your FCDPro+ work with CubicSDR and SDR++.  However, I would recommend you look up the thread where you modify the period (divisor for the sample rate).  You can look up that thread here: https://github.com/pothosware/SoapyFCDPP/issues/17.

FYI, this was on an Ubuntu system.
